hack/bicep-types-radius/generated/applications/applications.core/2023-10-01-preview/types.json

```diff
@@ -1854,10 +1854,17 @@
       },
       "env": {
         "type": {
-          "$ref": "#/151"
+          "$ref": "#/153"
         },
         "flags": 0,
         "description": "The environment variables injected during Terraform Recipe execution for the recipes in the environment."
+      },
+      "envSecrets": {
+        "type": {
+          "$ref": "#/154"
+        },
+        "flags": 0,
+        "description": "Environment variables containing sensitive information can be stored as secrets. The secrets are stored in Applications.Core/SecretStores resource."
```

Comment:
existing types.json has slightly different format/casing.  checking once that it is expected
```
 "envSecrets": {
          "Type": 158,
          "Flags": 0,
          "Description": "Environment variables containing sensitive information can be stored as secrets. The secrets are stored in Applications.Core/SecretStores resource."
        }
      }

Reply:
yes, that's expected. the generated format that the compiler expects is different than the current version we use